src/app/zap-templates/zcl/data-model/chip/sample-mei-cluster.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Sample MEI</name>
+    <code>0xFFF1FC20</code>
+    <define>SAMPLE_MEI_CLUSTER</define>
+    <description>The Sample MEI cluster showcases a cluster manufacturer extensions</description>
+    <client tick="false" init="false">true</client>
```

Comment:
what are tick and init here?

Reply:
I've seen this in other XML files, such as `level-control-cluster.xml`. My understanding is that this would be controlling whether ember calls its init and periodic ticks, which I don't care and don't want to.

But I'm not positive this is wired up in ember as it exists in Matter. It could be some legacy dead code from zigbee. I'll try removing and seeing if it breaks